### PR TITLE
[controlsd] Improved lateral controls in Toyota Prius 2017

### PR DESCRIPTION
**Pull request by nuwandavek**
- Set deadzone back to 0
- Set lag from data, by eps fw versions
- Set torqued offline values by fw versions

### CODE REVIEW

**Comment by nuwandavek** [comment]
@adeebshihadeh I'm passing a pseudo platform to the `configure_torque_tune` function. Is there a better pattern to use?

File: selfdrive/car/torque_data/params.yaml [hunk 1]
```diff
@@ -77,7 +77,8 @@ TOYOTA HIGHLANDER 2020: [2.022340166827233, 1.6183134804881791, 0.14592306380054
 TOYOTA HIGHLANDER HYBRID 2018: [1.752033, 1.6433903296845025, 0.144600]
 TOYOTA HIGHLANDER HYBRID 2020: [1.901174, 2.104015182965606, 0.14447040132184993]
 TOYOTA MIRAI 2021: [2.506899832157829, 1.7417213930750164, 0.20182618449440565]
-TOYOTA PRIUS 2017: [1.746445, 1.5023147650693636, 0.151515]
+TOYOTA PRIUS 2017: [1.60, 1.5023147650693636, 0.151515]
+TOYOTA PRIUS 2017 x060: [1.746445, 1.5023147650693636, 0.17]
```
Comment (adeebshihadeh) [inline selfdrive/car/torque_data/params.yaml:81]: this shouldn't be here. i'll make the tests fail too

File: selfdrive/car/interfaces.py [hunk 2]
```diff
@@ -109,7 +109,7 @@ def get_steer_feedforward_function(self):
   def torque_from_lateral_accel_linear(lateral_accel_value, torque_params, lateral_accel_error, lateral_accel_deadzone, friction_compensation):
     # The default is a linear relationship between torque and lateral acceleration (accounting for road roll and steering friction)
     friction_interp = interp(
-      apply_deadzone(lateral_accel_error, lateral_accel_deadzone),
+      apply_slack(lateral_accel_error, lateral_accel_deadzone),
```
Comment (adeebshihadeh) [inline selfdrive/car/interfaces.py:112]: can you rename all the deadzones to match this?

**Review by adeebshihadeh — COMMENTED** [review]
Can you give the PR title (and commit message) a more descriptive name? Something like "Toyota: improved lateral tuning for some 2017 Prius"

File: selfdrive/car/torque_data/params.yaml (hunk 1 above)
Comment (sshane) [inline selfdrive/car/torque_data/params.yaml:81]: this isn't used anywhere?

Comment (nuwandavek) [reply selfdrive/car/torque_data/params.yaml:81]: https://github.com/commaai/openpilot/pull/26626

**Comment by Edison-CBS** [comment]
May I ask how to use offline data to reset torqued offline values for TOYOTA PRIUS v 2017 by my own fork?

**Pull request merged**